components/ui/Title.jsx

```diff
@@ -0,0 +1,3 @@
+const Title = ({ name }) => <h3 className="text-2xl leading-7 text-primaryGray-300 -tracking-tighter">{ name }</h3>
```

Comment:
The font size change according to the screen the text is being displayed on, make sure you take it into consideration.